src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (firstVar) => {
+    count++;
+
+    const typeCondition = count < 3 || count % 2 === 1;
+
+    if (typeCondition) {
+      return (secondVar) => firstVar + secondVar;
+    } else {
+      return () => 'Bzzz... Error!';
+    }
```

Comment:
else if redundant after return statement

Reply:
yes else here is not necessary, will be removed 👍